devops/jobs/CreateCeleryDashboard.groovy

```diff
@@ -0,0 +1,113 @@
+            configuration.environments.each { environment ->
+                dslFactory.job(extraVars.get("FOLDER_NAME","Monitoring") + "/create-celery-dashboard-${environment}-${deployment}") {
+
+                    wrappers common_wrappers
+                    /* Only keep the builds for one day since it runs every minute.
+                       Reduce number of builds kept from ~10,000 to ~1000
+                    */
+                    logRotator {
+                        daysToKeep(1)
+                    }
+
+                    wrappers {
+                        credentialsBinding {
+                            def variable = "redis-monitoring-${deployment}-role-arn"
+                            string('ROLE_ARN', variable)
+                            file('AWS_CONFIG_FILE','tools-edx-jenkins-aws-credentials')
+                        }
+                    }
+
+                    def config_internal_repo = "git@github.com:edx/${deployment}-internal.git"
+                    def config_secure_repo = "git@github.com:edx-ops/${deployment}-secure.git"
+
+                    extraVars['CONFIGURATION_INTERNAL_REPO'] = config_internal_repo
+                    extraVars['CONFIGURATION_SECURE_REPO'] = config_secure_repo
+
+                    if (environment == 'prod'){
+                         extraVars['NOTIFY_ON_FAILURE'] = 'tools-edx-jenkins-alert@edx.opsgenie.net'
+                    }
+                    else{
+                        extraVars['NOTIFY_ON_FAILURE'] = 'devops+non-critical@edx.org'
+                    }
+
+                    properties {
+                        rebuild {
+                            autoRebuild(false)
+                            rebuildDisabled(false)
+                        }
+                    }
+
+                    throttleConcurrentBuilds {
+                        maxPerNode(0)
+                        maxTotal(0)
+                    }
+
+                    triggers {
+                        cron("H/10 * * * *")
+                    }
+
+                    parameters common_parameters(extraVars)
+
+                    multiscm common_multiscm(extraVars)
+
+                    environmentVariables {
+                        env('ENVIRONMENT', environment)
+                        env('DEPLOYMENT', deployment)
+                        env('AWS_DEFAULT_REGION', configuration.get('aws_region'))
+                    }
+
+                    steps {
+                        virtualenv {
+                            pythonName('System-CPython-3.5')
+                            nature("shell")
+                            systemSitePackages(false)
+
+                            command(
+                                dslFactory.readFileFromWorkspace("devops/resources/create_celery_dashboard.sh")
+                            )
+
+                        }
+                    }
+
+                    // publishers {
```

Comment:
Do you have an opinion on having commented code in a repo / could we remove this or do we plan on uncommenting it once we merge it?  (Not sure if there is a standard workflow or not for this repo)